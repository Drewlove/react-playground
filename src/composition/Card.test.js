import React from 'react'; 
import ReactDOM from 'react-dom'; 
import renderer from 'react-test-renderer'
import Card from './Card'; 

describe('card test', ()=> {
    it('renders without crashing', ()=> {
        const div = document.createElement('div'); 
    
        ReactDOM.render(<Card />, div); 
        ReactDOM.unmountComponentAtNode(div); 
    })
    
    it('renders UI as expected', ()=> {
        const tree = renderer
        .create(<Card title='new title' content='new content'/>)
        .toJSON(); 
        expect(tree).toMatchSnapshot(); 
    })
})
