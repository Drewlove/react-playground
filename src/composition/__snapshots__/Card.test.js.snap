// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card test renders UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    new title
  </h3>
  <p>
    new content
  </p>
</div>
`;

exports[`renders UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    new title
  </h3>
  <p>
    new content
  </p>
</div>
`;
